### ISSUE TITLE
reusing the source returned by PaymentIntent

### ISSUE THREAD

**Issue by sepehr-laal** [stripe/stripe-payments-demo#56]
I am trying to reuse the `source` that returns over PaymentIntent object, however it seems like it is not a chargeable source. The server also never receives the `source.chargeable` webhook mentioned in [best practices](https://stripe.com/docs/sources/best-practices).

How do I reuse the source created within this demo?

**Comment by thorsten-stripe** [comment]
The easiest way to achieve this is by specifying `save_payment_method ` within `stripe.handleCardPayment`[0]. Note that for this to work you will have to have set a customer ID on the PaymentIntent[1].

When using card sources with PaymentIntents no `source.chargeable` event is emitted, as card sources are immediately chargeable.

Please note that this issue tracker is for bugs or issues directly related to this project, and not for general support. Please feel free to reach out to https://support.stripe.com/email and we would be glad to help further!

[0] https://stripe.com/docs/stripe-js/reference#stripe-handle-card-payment
[1] https://stripe.com/docs/api/payment_intents/create#create_payment_intent-customer

**Comment by sepehr-laal** [comment]
@thorsten-stripe thank you! sorry if I reported in the wrong issue tracker, I just started working off of a fork of this project, that's why I asked here.

My use case is that, I want to subscribe my users to a plan immediately after they pay their activation fee. So I attach the customer to my PaymentIntent and used `save_payment_method` that you mentioned. I also pass the selected plan id in the `description` field of my PaymentIntent so I can look it up later when received over webhook.
